src/client/common/process/interpreterActivation.ts

```diff
@@ -352,6 +302,132 @@ export class EnvironmentActivationService implements IEnvironmentActivationServi
 
         return promise;
     }
+    public async getActivatedEnvVarsUsingActivationCommands(resource: Resource, interpreter: PythonEnvironment) {
```

Comment:
Existing code, just made this public for testing, not sure why it shows up as different